### PR TITLE
Integrate with Linux file managers

### PR DESCRIPTION
**Pull request by illuminace**
Associates JD-GUI with Java .class files and .jar archives. Tested
with Nautilus and Thunar.

### CODE REVIEW

File: src/linux/resources/jd-gui.desktop [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Comment=Java Decompiler JD-GUI
 Terminal=false
 Name=JD-GUI
-Exec=java -jar /opt/jd-gui/jd-gui.jar
+Exec=java -jar /opt/jd-gui/jd-gui.jar %F
 Type=Application
 Icon=jd-gui
+MimeType=application/java-archive;application/java-vm
```
Comment (samrocketman) [inline src/linux/resources/jd-gui.desktop]: According to an [original document by Sun](https://docs.oracle.com/cd/E19146-01/819-2630/abumo/index.html).  The mimetype `application/java-vm` was reserved just for `*.class` files.  Therefore, this should read:

```desktop
MimeType=application/java-archive
```

**Comment by samrocketman** [comment]
`%F` is for opening one or more files according to the [desktop entry spec](http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s06.html).  Looks OK to me other than the mimetype.

File: src/linux/resources/jd-gui.desktop (hunk 1 above)
Comment (samrocketman) [inline src/linux/resources/jd-gui.desktop]: I take my former comment back:  I misunderstood that Mimetype was supported files the application could open according to the [desktop entry spec](http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s05.html).

**Comment by samrocketman** [comment]
Looks OK to me including the original supported mimetypes.

**Comment by emmanue1** [comment]
Added: https://github.com/java-decompiler/jd-gui/commit/b1d04211ec31cafa9d2e9e7cbaa0c8a1d0bd417f

**Pull request closed without merging**